src/wallet/rpcdump.cpp

```diff
@@ -377,12 +419,9 @@ UniValue dumpwallet(const UniValue& params, bool fHelp)
             HelpExampleCli("dumpwallet", "\"test\"") + HelpExampleRpc("dumpwallet", "\"test\""));
 
     LOCK2(cs_main, pwalletMain->cs_wallet);
-
```

Comment:
The removal of the blank lines in this routine is inconsistent with the rest of the file's style.